pages/dao/[dao]/treasury/tokens/index.ts

```diff
@@ -2,35 +2,35 @@ import Tokens, {
   TokensPageProps,
 } from 'pages/dao/[dao]/treasury/tokens/TokensPage';
 import { SputnikHttpService } from 'services/sputnik';
-import { getChartData } from 'features/treasury/helpers';
 import { GetServerSideProps } from 'next';
+import { CookieService } from 'services/CookieService';
+import { ACCOUNT_COOKIE } from 'constants/cookies';
 
 export const getServerSideProps: GetServerSideProps<TokensPageProps> = async ({
+  req,
   query,
 }) => {
   const daoId = query.dao as string;
 
-  const [dao, receipts, policyAffectsProposals] = await Promise.all([
-    SputnikHttpService.getDaoById(daoId),
+  CookieService.initServerSideCookies(req?.headers.cookie || null);
+
+  const account = CookieService.get<string | undefined>(ACCOUNT_COOKIE);
```

Comment:
Looks like this piece of code is being duplicated. Maybe, there is sense to move it to a separate method and reuse?

Reply:
yes, in next iteration i want to extract entire page init method into separate entity